### DISCUSSION TITLE
20241108 Bugs

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1801] [Bug Reports]
DAB-Scan seems to work , but no DAB-Icon in Library .
WIFI Static IP will not connect , only DHCP .
No more 5 GHz , only 2.4 GHz .

**Comment by rern** [comment]
For DAB, please try:
```sh
sed -i "/mode .label/ a\
        \$( '.mode.dabradio' ).toggleClass( 'hide', C.dabradio === 0 );
" /srv/http/assets/js/function.js

/srv/http/bash/cmd.sh cachebust
```

**Reply by kan-ibal** [reply]
This fixed DAB-Icon in Library but still DAB toggle switch is broken in Settings. Switch is frozen in ON position and was working before update.

**Comment by hjfisch** [comment]
![Screenshot 2024-11-09 112002](https://github.com/user-attachments/assets/bc9c62c5-2b50-4f2e-a0a0-e13bcd0c95bd)
![Screenshot 2024-11-09 112026](https://github.com/user-attachments/assets/8165521f-3f93-46cf-9443-32824d0b7b0f)
![Screenshot 2024-11-09 112047](https://github.com/user-attachments/assets/2dd3b551-d93e-460d-a960-659363d1bf34)
Now the dab-icon is back and shows over 70000 stations , but I've found only about 50 . In the folder is nothing .
Wlan is still broken , so I will return to 241025 .

**Comment by hjfisch** [comment]
B.t.w. 5 GHz Wlan and Static IP is broken too in 241025 . It is only working if you come from an earlier Version with activated Wlan . If you try to change the Adress it will not work anymore .
![Screenshot 2024-11-09 140302](https://github.com/user-attachments/assets/0d8ae38a-1e69-42cf-bb7f-34046bb5b442)
Not Green but working .
Now I've managed Wlan with 5GHz to work with the latest Version and DHCP (Nogo with Static ), but sadly not DAB .
![Screenshot 2024-11-09 145611](https://github.com/user-attachments/assets/a4911acf-fc45-4f0c-9b98-b1e5836d1fa2)

**Comment by rern** [comment]
Please post the output of: `dab-scanner-rtlsdr -C 5A`

**Reply by hjfisch** [reply]
[dab-Scan.txt](https://github.com/user-attachments/files/17690955/dab-Scan.txt)
B.t.w. "srv/http/dabradio" Folder is empty

**Reply by rern** [reply]
Try: 
- `mkdir /srv/http/data/dabradio`
- Then Features > DAB Radio > setting - Scan

**Comment by hjfisch** [comment]
Things are getting better .
First I've installed DAB-Radio plugin on 241025 .
Than update to 20241108 .
sed -i "/mode .label/ a\
        \$( '.mode.dabradio' ).toggleClass( 'hide', C.dabradio === 0 );
" /srv/http/assets/js/function.js

/srv/http/bash/cmd.sh cachebust
No need to "mkdir /srv/http/data/dabradio" , because it was already there.
Did a scan and now it looks like this :
![Screenshot 2024-11-10 161756](https://github.com/user-attachments/assets/953b2bcc-626b-4755-b40a-e0c578a6c8f7)
![Screenshot 2024-11-10 161848](https://github.com/user-attachments/assets/2b885bdb-be67-4d20-9b35-2dc73751a062)
All scanned stations are there but will not play .
When I press play it will directly return to stop after 1 second .

**Reply by rern** [reply]
Try: `/srv/http/bash/settings/player.sh ffmpeg`

**Comment by hjfisch** [comment]
Was activated , no change .

**Comment by hjfisch** [comment]
Did a new update today . Now it runs for about 10 to 20 seconds , but without sound .

**Comment by hjfisch** [comment]
Tried a better Antenna with external power today .
Receiving 74 stations now , but still stopping after 10 to 20 seconds with no sound .

**Reply by ocram76** [reply]
Hi, I also have the same problem, did you solve it, could you possibly tell me how you did it? Thank you

**Reply by hjfisch** [reply]
No , I've removed the stick .
